assets/js/index.js

```diff
@@ -0,0 +1,366 @@
+// Global declaration of variables
+
+const recentSearchesContainer = $("#recent-searches-container");
+const weatherInfoContainer = $("#weather-info-container");
+const searchForm = $("#search-form");
+
+// reususable modular function to read data from Local Storage
+const readFromLocalStorage = (key, defaultValue) => {
+	// get from LS using key name
+	const dataFromLS = localStorage.getItem(key);
+
+	// Parse data from LS
+	const parsedData = JSON.parse(dataFromLS);
+
+	if (parsedData) {
```

Comment:
This would be nice as a ternary statement:

`parsedData ? parsedData : defaultValue`